### PR TITLE
Add alternative palettes

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

- Add alternative palettes
- Create a QA page to vizualise colors

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2280/storybook

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/6460/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2280/storybook

File: packages/scss/src/commons/config.scss [hunk 1]
```diff
@@ -87,11 +100,90 @@ $grey: (
...
 ) !default;
 
-$navigation: (
-	text: #ffffff,
-	700: #313972,
-	800: #192157,
-	900: #12183F,
+// Colors : Decoratives
+
+$kiwi: (
```
Comment (BertrandPodevin) [inline packages/scss/src/commons/config.scss:105]: `$kiwi` is triplicate : l105, l153 & 159

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2280/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/6467/ui-diff)

**Review by BertrandPodevin — APPROVED** [review]

**Pull request merged**